data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -0,0 +1,141 @@
+
+.. This is the label that can be used for cross referencing this file.
+.. Recommended title label format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. Each section should include a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link and explicit title using the syntax :ref:`link text <label-name>`.
+.. A warning will alert you of identical labels during the linkcheck process.
+
+
+.. _Data-Access-Analysis-Tools-NB-Intro:
+
+#######################################
+Introduction to the RSP Notebook Aspect
+#######################################
+
+Most RSP users will find Jupyter Notebooks to be the most efficient and powerful way to interact with the DP0.1 data set.
+For DP0.1, Jupyter Notebooks will be the only way to access images. 
+
+
+.. _NB-Intro-Login:
+
+Log In and Out of JupyterLab
+============================
+
+From the RSP landing page at `data.lsst.cloud <https://data.lsst.cloud/>`_ click on the central panel for Notebooks.
+
+**Software Version and Server Size:**
+The first page offers a choice of software environment version (left) and server size (right), as in the figure below.
+Most users will choose the recommended software version and a medium server size. 
+
+.. figure:: /_static/RSP_NB_select_a_server.png
+    :width: 400
+    :name: RSP_NB_select_a_server
+
+    Most users will choose the recommended software version and a medium size.
+    
+The term "image" atop the left box refers to a "Docker image" that defines the software packages and their versions which will be automatically loaded in the server environment.
+The "recommended" image will be updated on a monthly basis during DP0.1 to encourage users to adapt to using software that is an active development, and to benefit from the bug fixes and updates made by Rubin staff.
+Older images will remain accessible to users.
+
+RSP users who are doing a lot of image processing might need to select a large server, and those who are working with small subsets of catalog data can use a small server.
+
+**Start the Server:**
+Pressing the orange start button to start the server returns this page with a blue progress bar.
+
+.. figure:: /_static/RSP_NB_progress_bar.png
+    :width: 400
+    :name: RSP_NB_progress_bar
+
+    Be patient. Sometimes it takes a couple of minutes to start a server.
+
+**JupyterLab Navigation:**
+The JupyterLab landing page in the figure below is the launch pad for all JupyterLab functionality (e.g., Notebook, Terminal).
+Return to this launch pad at any time by clicking the plus symbol at upper left.
+For DP0.1, only Python Notebooks and the terminal interface are supported (i.e., not the Console), and users are not able to access Portal queries from a Jupyter Notebook.
```

Comment:
* I'm not sure what you mean by the Console not being supported.  It works and a perfectly reasonable way to interact with things, if people prefer a more linear approach.  Are you just saying the CET isn't intending to support questions about the Console?
* I'm also not clear on what the statement about not being able to access Portal queries is trying to accomplish.  It's true that you can't import a full Portal search experience into a notebook, but that seems to be too negative.  I think the statement could read better as "TAP queries executed in the Portal can be rerun with the TAP client to access the same data in either place"